### ISSUE TITLE
Vollständigkeit bei seperater Erfassung fehlerhaft

### ISSUE THREAD

**Issue by synchronisator** [osmberlin/osm-parking-processing#69]
Wenn an einem Straßenabschnitt die Parkplätze separat erfasst werden, dann werden Straßenteile die länger sind als die erfasste Parkfläche als "unvollständig" markiert. Das ist nicht korrekt, hier sollte keine Markierung oder eine "Parken verboten" Markierung verwendet werden.
Beispiel:
![grafik](https://user-images.githubusercontent.com/10772789/226100161-cbf441f6-9bcb-4f2e-b8bc-0c9a602140a1.png)

https://www.openstreetmap.org/#map=19/52.40722/9.78692

**Comment by SupaplexOSM** [comment]
Stimmt, bei einer Straßenseite mit `parking:*=separate` sollte davon ausgegangen werden, dass keine Daten mehr fehlen.

Hier das Straßenstück aus dem oben gezeigten Beispiel: [OSM](https://www.openstreetmap.org/way/1109616277) |[Parkraum-Vector-Tiles](https://radverkehrsatlas.de/regionen/parkraum?lat=52.40725&lng=9.78706&zoom=18.3&theme=parking&bg=default&config=!(i~parking~topics~!(i~parking~a~~s~!(i~default~a~_F)(i~presence~a)(i~raw~a~_F))(i~parkingPoints~a~_F~s~!(i~default~a))(i~parkingAreas~a~~s~!(i~default~a)(i~position-separate~a~_F))(i~parkingDebug~a~_F~s~!(i~default~a))(i~parkingStats~a~_F~s~!(i~default~a~~f~!(i~admin*_level~o~!(i~*4~a~_F)(i~*9~a)(i~*10~a~_F)))(i~length~a~_F~f~!(i~admin*_level~o~!(i~*4~a~_F)(i~*9~a)(i~*10~a~_F))))(i~landuse~a~_F~s~!(i~default~a)))~)

**Comment by gislars** [comment]
Was genau ist in diesem Fall eine Straßenseite? Vielleicht hat sich auch durch andere Updates etwas geändert. Aber nehmen wir an, ein Straßenabschnitt von Kreuzung zur nächsten Kreuzung hat parking:*=separate, dann sollte dieser gesamte Abschnitt "OK" sein, auch wenn die separat gemappte Prkfläche kleiner/kürzer als der Straßenabschnitt ist?

**Comment by SupaplexOSM** [comment]
Ich glaube auch, hier hat sich was verändert. Die oben gezeigte Situation kann ich jedenfalls nicht mehr reproduzieren. In jedem Fall sollte eine Straßenseite (also `left`, `right` oder `both`), die mit `separate` getaggt ist, sowohl in der Vollständigkeitsansicht als auch in der Statistik in ihrer vollen Länge als "vollständig mit Daten ausgestattet" betrachtet werden.
